dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityOperationParamsMapper.java

```diff
@@ -114,6 +125,8 @@ public TrackedEntityQueryParams map(
     validateTrackedEntityAttributeFilters(
         program, requestedTrackedEntityType, operationParams, orgUnits, params);
 
+    validatePagination(pageParams);
```

Comment:
Maybe do this early as this seems like one of the cheaper checks we do. So we return early in the scenario and don't fetch all these other things.